namespaces/cloud-platform-test-1.k8s.integration.dsd.io/fluentd-es/kube-fluentd-es.yaml

```diff
@@ -0,0 +1,126 @@
+#This file deploys a 'logging' namespace and a Fluentd DaemonSet, along with the supporting RBAC.
+#This file is linked to the following issue https://github.com/ministryofjustice/cloud-platform/issues/283
+
+---
+apiVersion: v1
```

Comment:
Can you please split off the namespace in its own file, called `00-namespace.yaml`?
It ensures `kubectl` will apply this file before the rest, otherwise it will fail on the first apply.